lib/ohai/plugins/linux/platform.rb

```diff
@@ -113,6 +129,15 @@ def os_release_file_is_cisco?
       # no way to determine platform_version in a rolling release distribution
       # kernel release will be used - ex. 3.13
       platform_version `uname -r`.strip
+    elsif (os_release_info = os_release_file_is_cisco?)
+      # Cisco platform not based on known distro above
+      if os_release_info['ID'] == 'nexus'
+        platform 'nexus'
+        platform_family 'cisco-wrlinux'
+      elsif os_release_info['ID_LIKE'] =~ /cisco-wrlinux/
```

Comment:
I don't see a test for this case. Is this legit? Can we get a test and a comment with what platform this might catch?